src/test/java/HandleGetRequestTest.java

```diff
@@ -18,48 +18,48 @@
 
   protected void setUp() {
     MockFileIO mockFileIO = new MockFileIO();
-    handler = new HttpHandlerSelector(mockFileIO);
+    getRequest = new HandleGetRequest(mockFileIO);
   }
 
   public void testGenerateOkResponseDefaultPath() {
     request = new HttpRequest("GET / HTTP/1.1");
-    response = handler.generateResponse(request);
+    response = getRequest.generateResponse(request);
     assertEquals("HTTP/1.1 200 OK", response.responseLine());
   }
 
   public void testGenerateOkResponseValidFile() {
     request = new HttpRequest("GET /something.txt HTTP/1.1");
-    response = handler.generateResponse(request);
+    response = getRequest.generateResponse(request);
     assertEquals("HTTP/1.1 200 OK", response.responseLine());
   }
 
   public void testGenerateNotFoundResponseforInvalidFile() {
     request = new HttpRequest("GET /throwIOException HTTP/1.1");
-    response = handler.generateResponse(request);
+    response = getRequest.generateResponse(request);
     assertEquals("HTTP/1.1 404 Not Found", response.responseLine());
   }
 
   public void testContentTypeIsTextHtmlForTxtFile() {
     request = new HttpRequest("GET /something.txt HTTP/1.1");
-    response = handler.generateResponse(request);
+    response = getRequest.generateResponse(request);
```

Comment:
I was suggesting that maybe a class can be pulled out for this, but in retrospect, maybe it would be overkill.

Not sure.

I guess just leave it for now.